tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -0,0 +1,146 @@
+import uuid
+from datetime import datetime, timedelta
+from typing import Any, Dict
+
+import pytest
+from django.utils import timezone
+
+from sentry.monitors.consumers.check_in import _process_message
+from sentry.monitors.models import (
+    CheckInStatus,
+    Monitor,
+    MonitorCheckIn,
+    MonitorEnvironment,
+    MonitorStatus,
+    MonitorType,
+    ScheduleType,
+)
+from sentry.testutils import TestCase
+from sentry.utils import json
+
+
+class MonitorConsumerTest(TestCase):
```

Comment:
Could be worth adding one test that tests submitting to the strategy as well, similar to https://github.com/getsentry/sentry/blob/f5ab3c97bf6442a8a9cf0b6f835d2a68eaf17736/tests/sentry/snuba/test_query_subscription_consumer.py#L105-L123

Reply:
ah so that's how you make a kafka payload for testing...

Reply:
was just going off `occurrence_consumer`